docs/docsite/rst/playbook_guide/playbooks_tags.rst

```diff
@@ -334,8 +334,8 @@ Once you have added tags to your tasks, includes, blocks, plays, roles, and impo
 :ref:`ansible-playbook` offers five tag-related command-line options:
 
 * ``--tags all`` - run all tasks, ignore tags (default behavior)
-* ``--tags [tag1, tag2]`` - run only tasks with either the tag ``tag1`` or the tag ``tag2``
-* ``--skip-tags [tag3, tag4]`` - run all tasks except those with either the tag ``tag3`` or the tag ``tag4``
+* ``--tags tag1, tag2`` - run only tasks with either the tag ``tag1`` or the tag ``tag2``
+* ``--skip-tags tag3, tag4`` - run all tasks except those with either the tag ``tag3`` or the tag ``tag4``
```

Comment:
Is that space actually valid? I would guess it won't work with that space, since then `tar2` resp. `tag4` wouldn't be part of the argument for `--tags` resp. `--skip-tags`.
```suggestion
* ``--tags tag1,tag2`` - run only tasks with either the tag ``tag1`` or the tag ``tag2``
* ``--skip-tags tag3,tag4`` - run all tasks except those with either the tag ``tag3`` or the tag ``tag4``
```